diff -uNr acarsdec/CMakeLists.txt acarsdec.af/CMakeLists.txt
--- acarsdec/CMakeLists.txt	2024-09-21 21:56:52.229139415 +0000
+++ acarsdec.af/CMakeLists.txt	2024-09-21 21:55:56.278659745 +0000
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.2)
 project (acarsdec C)
 
-add_compile_options(-Ofast -march=native)
+add_compile_options(-Ofast)
 
 add_executable(acarsdec acars.c  acarsdec.c  cJSON.c  label.c  msk.c  output.c netout.c fileout.c )
 
diff -uNr acarsdec/debian/changelog acarsdec.af/debian/changelog
--- acarsdec/debian/changelog	1970-01-01 00:00:00.000000000 +0000
+++ acarsdec.af/debian/changelog	2024-09-21 21:55:42.248540088 +0000
@@ -0,0 +1,5 @@
+acarsdec (3.7-2) bookworm; urgency=low
+
+  * Remove the -march=native flag from gcc.
+
+ -- Stanislav Lechev [LZ2SLL] <af@0xaf.org>  Sun, 22 Sep 2024 00:07:00 +0100
+
+acarsdec (3.7-1) bookworm; urgency=low
+
+  * Packaging by LZ2SLL (Thanks to Jakob Ketterl <jakob.ketterl@gmx.de>)
+
+ -- Stanislav Lechev [LZ2SLL] <af@0xaf.org>  Fri, 05 Jul 2024 22:05:38 +0000
diff -uNr acarsdec/debian/control acarsdec.af/debian/control
--- acarsdec/debian/control	1970-01-01 00:00:00.000000000 +0000
+++ acarsdec.af/debian/control	2024-09-21 21:55:42.248540088 +0000
@@ -0,0 +1,24 @@
+Source: acarsdec
+Section: hamradio
+Priority: optional
+Maintainer: Stanislav Lechev [LZ2SLL] <af@0xaf.org>
+Rules-Requires-Root: no
+Build-Depends:
+ debhelper-compat (= 13),
+ cmake,
+ libacars-dev,
+ libsndfile1-dev,
+ pkg-config,
+Standards-Version: 4.6.2
+Homepage: https://github.com/TLeconte/acarsdec
+Vcs-Browser: https://github.com/TLeconte/acarsdec
+Vcs-Git: https://github.com/TLeconte/acarsdec.git
+
+Package: acarsdec
+Architecture: any
+Depends:
+ ${shlibs:Depends},
+ ${misc:Depends},
+Description: ACARS SDR decoder
+ Acarsdec is a multi-channels acars decoder with built-in rtl_sdr, airspy front
+ end or sdrplay device.
diff -uNr acarsdec/debian/copyright acarsdec.af/debian/copyright
--- acarsdec/debian/copyright	1970-01-01 00:00:00.000000000 +0000
+++ acarsdec.af/debian/copyright	2024-09-21 21:55:42.248540088 +0000
@@ -0,0 +1,34 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: <url://example.com>
+Upstream-Name: acarsdec
+Upstream-Contact: https://github.com/TLeconte/acarsdec
+
+Files:
+ *
+Copyright:
+ 2014-2018, Thierry Leconte
+License: LGPL-2
+
+Files:
+ debian/*
+Copyright:
+ 2024 Stanislav Lechev [LZ2SLL] <af@0xaf.org>
+License: GPL-2.0+
+
+License: LGPL-2
+ This package is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+ .
+ This package is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <https://www.gnu.org/licenses/>
+Comment:
+ On Debian systems, the complete text of the GNU General
+ Public License version 2 can be found in "/usr/share/common-licenses/GPL-2".
+
diff -uNr acarsdec/debian/rules acarsdec.af/debian/rules
--- acarsdec/debian/rules	1970-01-01 00:00:00.000000000 +0000
+++ acarsdec.af/debian/rules	2024-09-21 21:55:42.248540088 +0000
@@ -0,0 +1,3 @@
+#!/usr/bin/make -f
+%:
+	dh $@
diff -uNr acarsdec/debian/source/format acarsdec.af/debian/source/format
--- acarsdec/debian/source/format	1970-01-01 00:00:00.000000000 +0000
+++ acarsdec.af/debian/source/format	2024-09-21 21:55:42.248540088 +0000
@@ -0,0 +1 @@
+3.0 (quilt)
